src/migrations/migrations/deleteNFTsReactQueryCache.ts

```diff
@@ -0,0 +1,17 @@
+import { Migration, MigrationName } from '../types';
+import store from '@/redux/store';
+import { queryClient } from '@/react-query';
+import { nftsQueryKey } from '@/resources/nfts';
+
+export function deleteNFTsReactQueryCache(): Migration {
+  return {
+    name: MigrationName.deleteNFTsReactQueryCache,
+    async migrate() {
+      const { wallets } = store.getState().wallets;
```

Comment:
when ive tried to get wallet state here during migrate i usually dont have any wallets in state yet 

you should be able to just increment the persister version on the query key and get this 4 free

Reply:
good call